yb-voyager/src/transform/transform.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package transform
+
+import (
+	"fmt"
+	"slices"
+
+	pg_query "github.com/pganalyze/pg_query_go/v6"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/query/queryparser"
+)
+
+/*
+	There can be various transformation possible on the exported schema
+
+	- Merge Constraints into CREATE TABLE															[DONE]
+	- Drop/Rename Objects																			[Future]
+	- Drop/Rename columns: Voyager doesn't support export data for a set of datatypes,				[Future]
+		transformation can be applied to remove those columns from CREATE
+	- Datatype transformation: Voyager doesn't support export data for a set of datatypes,			[Future]
+		transformation can be applied to change those datatypes to supported ones
+	- Rename Table Partitions: In Oracle(ora2pg) case, we might need to rename table partitions		[Future]
+		to match the naming convention of the target database or retain original names.
+
+*/
+
+// In future we can convert this to interface and have separate implementations for each transformation.
+// But for now, kept it simple as a struct with methods
+type Transformer struct {
```

Comment:
I'm not focusing too much on the design now. Looks fine for now! 
Once we have more cases to transform, we can re-visit